src/main/java/runge/worldcities/WorldCitiesTests.java

```diff
@@ -0,0 +1,45 @@
+package runge.worldcities;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.junit.Test;
+
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+import static junit.framework.TestCase.*;
+
+public class WorldCitiesTests
+{
+    @Test
+    public void getClosestCity() throws IOException {
+        //given
+        WorldCities worldCities = new WorldCities();
+        double givenLat = 40.7128;
+        double givenLon = -74.0060;
+
+        //when
+        String closestCity = worldCities.getClosestCity(givenLat, givenLon);
+
+        //then
+        assertEquals(closestCity, "New York");
+
+    }
+
+    @Test
+    public void CSVParser() throws IOException {
+        //given
+        File csvData = new File("worldcities.csv");
+        CSVParser parser = CSVParser.parse(csvData, Charset.defaultCharset(), CSVFormat.RFC4180);
```

Comment:
You didn't write CVSParser, there is no reason to test the code for it.

Reply:
but I need to test why my file isn't giving out the correct output ergo test